python/test/unit/language/test_core.py

```diff
@@ -4754,6 +4748,94 @@ def test_convert2d(M, N, src_layout, interm_layout, dst_layout, dtype, device):
     assert torch.equal(z, x)
 
 
+@pytest.mark.parametrize("shape", [
```

Comment:
This is a bit of a nit but it feels more like a regression test than a unit test to me. Can we add the emitIndices tests instead, so are more targeted

Reply:
I don't have an emit-indices test at the moment.  Writing one would require me to figure out by hand what are the correct indices to emit for 128x1, and because the code doesn't work, I have no idea if what I write would be correct.

I'm happy to skip adding this test if you think it's not worthwhile, although if and when we try to fix this bug we will definitely want to run this test.

FWIW if you object tot his test, there are a bunch of other tests in test_core that you probably want to delete as well. Maybe a good starter project for someone?

Reply:
Sure let's skip this for now. Make sure to keep the branch so nothing is lost. 

> FWIW if you object tot his test, there are a bunch of other tests in test_core that you probably want to delete as well. Maybe a good starter project for someone?

Yes I'm sure we could improve things there.